pkgs/applications/virtualization/ignite/default.nix

```diff
@@ -0,0 +1,75 @@
+{ lib
+, cni-plugins
+, buildGoModule
+, firecracker
+, containerd
+, runc
+, makeWrapper
+, fetchgit
+, git
+}:
+
+
+buildGoModule rec{
+  pname = "ignite";
+  version = "0.9.0";
+
+  # Using fetchgit because ignite use git informations in its building.
+  src = fetchgit {
```

Comment:
Please use fetchFromGitHub which also supports this option.

Reply:
Done! thanks!